jointDellyGermline/README.md

```diff
@@ -0,0 +1,33 @@
+<img src="https://oicr.on.ca/wp-content/themes/oicr/assets/img/logo.svg" width=25% height=25%>
+
+# DELLY Germline SV and CNV Calling  
+
+### Introduction
+DELLY performs germline structural variant (SV) and copy number variation (CNV) calling using a comprehensive tool designed for the detection of germline SVs and CNVs from next-generation sequencing data. Leveraging sophisticated algorithms, including discordant read pairs, split reads, and read depth analysis, DELLY identifies a wide range of genomic variations, including deletions, duplications, inversions, amplifications, and deletions. The Joint Delly Germline procedure takes in whole genome bam files and vcf files from DELLY, for a set of samples (ideally 20+ unrelated samples) and utilizes information from all samples to detect variants with greater sensitivity and accuracy.
+
+### Summary
+A predefined set of vcf, bam, and bai files are provided as input to the Joint Delly Germline workflow. This generates a joint SV vcf and a joint CNV vcf with information from all samples where a call is made in any one sample. 
+
+### Inputs  
+- VCF files: A set of VCF files that have been individually processed through DELLY for SV calling
+- BAM files: A set of whole genome bam files for all samples to be genotyped
+- BAI files: A set of bam index files to accompany all bam file inputs
+- genome resources: as specified in the WDL workflow
+
+### Procedure
+ 1. Identify the set of samples you want to perform germline SV and CNV calling on
+ 2. Construct an olive that will pull these files as inputs to the joint genotyping workflow. This workflow is available in vidarr stage. The olive can be set up as an .actnow. The olive on this page can be modified to specify the project, with other modifications to limit the input data as needed; for example we may only process cases that have been released up to a certain time point. An olive template "jointDellyGermline.shesmu" can be found in this repo.
```

Comment:
_This workflow is available in vidarr stage._ no longer relevant? 
perhaps a note that delly_normal_only olive is running in research